client/src/components/searchSeries/SearchSeries.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import SeriesFilters from '../seriesFilters/SeriesFilters';
+import seriesService from '../../services/series.service';
+import PaginationTable, { seriesColumnDetails } from '../paginationTable/PaginationTable';
+import { Col, Container, Row } from 'react-bootstrap';
+
+const SearchSeries = () => {
+    const [query, setQuery] = useState({ name: "", genres: [], statuses: [] });
+
+    const setFilters = (filters) => {
+        setQuery(filters);
+    }
+
+    const fetchData = async (currentPage = 1) => {
+        const { data, totalAmount } = await seriesService.getSeriesByFilters(query, currentPage);
+        return { content: data, totalElements: totalAmount };
+    };
+
+    return (
+        <Container fluid>
+            <Row>
+                <Col xs={4} md={2} className="mt-3">
```

Comment:
Styling:
Maybe add more `margin-left` to the filters, it seems too close to the left side of the page.
Also try to increase the spacing between filter options (name, genres, status)